src/internal/secrets-manager.ts

```diff
@@ -1,7 +1,7 @@
 import { JSONArray, JSONObject } from 'k6'
 import http, { RefinedResponse, ResponseType } from 'k6/http'
+import { uuidv4 } from 'https://jslib.k6.io/k6-utils/1.4.0/index.js'
 
-import { v4 as uuidv4 } from 'uuid'
```

Comment:
I'm not sure I'm getting what's the problem with this dependency. Can we just keep it without applying the change?

Reply:
Apologies, I should have been more explicit about this specifically 🙇🏻  

So far, k6-jslib-aws relied on the uuid package to generate uuids. We were using version 3.4.0, which is quite old, but had the advantage of not relying on node's `crypto.getRandomValues` API. This version is now deprecated, and the newer versions (from `7.0.0`, to `10.0.0`) all rely on the node API exclusively (as far as I can tell), meaning we can't rely on it in a k6 context anymore.

Hence, switching to k6-utils instead. (Which coincidentally makes me realize there is a `secure` flag to activate to use a cryptographic random source instead)

Reply:
Thanks for the explanation @oleiade, all clear now! Looks great! 👍🏻